Readme.md

```diff
@@ -242,6 +242,22 @@ for filtering out certain status codes. This means that you can place your code
 handling API errors like 400's in the same places as code for handling invalid
 responses.
 
+## Migration Guides
+
+This project follows [Semantic Versioning](http://semver.org).
+
+Please follow the appropriate guide below when **upgrading to a new major version** of Moya (e.g. 8.0 -> 9.0).
+
+### Upgrade from 8.x to 9.x
```

Comment:
Sure, I've moved it to its own `MigrationGuides.md` file within the `docs` folder. I think we should consider linking to the file when doing major releases within the changelog & release notes.